packages/taquito/src/context.ts

```diff
@@ -166,6 +168,29 @@ export class Context {
     }
   }
 
+  async getConfirmationPollingInterval() {
+    try {
+      const constants = await this.rpc.getConstants();
+      let confirmationPollingInterval = BigNumber.sum(constants.time_between_blocks[0], 
+        new BigNumber(constants.delay_per_missing_endorsement!)
+        .multipliedBy(Math.max(0, constants.initial_endorsers! - constants.endorsers_per_block))
+      );
+      
+      // Divide the polling interval by a constant 3
+      // to improvise for polling time to work in prod,
+      // testnet and sandbox enviornment.   
+      confirmationPollingInterval = confirmationPollingInterval.dividedBy(3);  
+      this.config.confirmationPollingIntervalSecond = confirmationPollingInterval.toNumber();
+      return this.config.confirmationPollingIntervalSecond;
+    } catch (exception) {
+      console.log(exception);
```

Comment:
Should we remove that? we don't want dapp builders seeing unexpected logging in their app

Reply:
This is fixed.